.env.dist

```diff
@@ -1,76 +1,9 @@
-###> Docker Compose Configuration ###
-COMPOSE_PROJECT_NAME=ergonode
-CONTAINER_REGISTRY_BASE=ergonode
-####< Docker Compose Configuration ###
-
-###> PostgreSQL Configuration ###
-POSTGRES_PASSWORD=123
-POSTGRES_USER=postgres
-EXPOSED_POSTGRES_PORT=5432
-POSTGRES_DB=postgres
-####< PostgreSQL Configuration ###
-
-###> App DB Configuration ###
-APP_USER=ergonode
-APP_USER_PASSWORD=123
-APP_DB=ergonode
-APP_TEST_DB=ergonode_test
-####< App DB Configuration ###
-
-###> Xdebug Configuration ###
-# Default xdebug config for php target php_final_with_xdebug
-# This ip can be get by using shell command:
-# ifconfig | grep -A1 docker0 | grep inet
-# on Mac OS X and Windows `host.docker.internal` can be set for autoresolving
-# if you have firewall you need enable listening on tcp port 9000
-# for ufw firewall you can use command like
-# sudo ufw allow 9000/tcp
-# XDEBUG_CONFIG=remote_enable=1 remote_host=host.docker.internal remote_connect_back=0
-
-# Default ide config for ./bin/console command
-# and phpstorm should be localhost or symfony.dev
-PHP_IDE_CONFIG=serverName=localhost
-####< Xdebug Configuration ###
-
-
-###> Nginx Configuration ###
-EXPOSED_NGINX_PORT=8000
-####< Nginx Configuration ###
-
-###> PHP Configuration ###
-PHP_FINAL_TARGET=php_final
-###< PHP Configuration ###
-
-###> Node Configuration ###
-EXPOSED_NODE_PORT=80
-###< Node Configuration ###
-
-###> Docs Configuration ###
-EXPOSED_DOCS_PORT=8002
-###< Docs Configuration ###
-
-###> Backend overrides ###
-APP_URL=http://nginx
-###< Backend overrides ###
-
-###> Rabbitmq Configuration ###
-EXPOSED_RABBITMQ_PORT=5672
-EXPOSED_RABBITMQ_WEB_PORT=15672
-RABBITMQ_DEFAULT_USER=ergonode
-RABBITMQ_DEFAULT_PASS=ergonode
-###< Rabbitmq Configuration ###
-
-###> Mailhog Configuration ###
-EXPOSED_MAILHOG_SMTP_PORT=1025
-EXPOSED_MAILHOG_WEB_PORT=8025
-###< Mailhog Configuration ###
-
-###> Worker ###
-WORKER_SLEEP=10
-IMPORT_WORKER_SLEEP=30
-EXPORT_WORKER_SLEEP=30
-WORKER_MEMORY_LIMIT=128M
-IMPORT_WORKER_MEMORY_LIMIT=512M
-EXPORT_WORKER_MEMORY_LIMIT=512M
-WORKER_TIME_LIMIT=3600
-###< Worker ###
+###> PHP container ###
+# For Windows and Mac OS X users
+PHP_IDE_CONFIG=host.docker.internal
```

Comment:
Fairly sure this should be as for Linux version `serverName=***`

Reply:
When i finish we can check because i find it on official XDebug site